src/kOS.Safe/Encapsulation/PrimitiveStructure.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace kOS.Safe.Encapsulation
+{
+    [kOS.Safe.Utilities.KOSNomenclature("Structure", KOSToCSharp = false)]
+    public abstract class PrimitiveStructure : Structure
+    {
+        public abstract object ToPrimitive();
+    }
+}
+
```

Comment:
The pattern of a `Structure` implementing a `ISerializableValue` would be repeated a few times so I thought it might make sense to have a common parent class.